src/pkg/backup/backup.go

```diff
@@ -117,6 +123,8 @@ func (b Backup) Headers() []string {
 		"ID",
 		"Status",
 		"Selectors",
+		"Size",
```

Comment:
@meain - I'm thinking that we should just call this BytesRead and BytesUploaded in the JSON output - since it's the most accurate description.

Reply:
I've renamed them to `BytesRead` and `BytesUploaded`. I've also removed them from cli listing keeping them only in the json output as discussed.